lib/amplitude/event.rb

```diff
@@ -0,0 +1,285 @@
+require 'json'
+
+module AmplitudeAnalytics
+  # Plan
+  class Plan
```

Comment:
`Plan` is only used by ampli. If we don't have a plan to support ampli soon. Feel like we can remove this